src/Core__Array.res

```diff
@@ -214,3 +214,12 @@ let findMap = (arr, f) => {
 }
 
 @send external at: (array<'a>, int) => option<'a> = "at"
+
+@val external of0: unit => array<'a> = "Array.of"
+@val external of1: 'a => array<'a> = "Array.of"
+
+let fromOption = opt =>
+  switch opt {
+  | None => of0()
```

Comment:
Can't this just return `[]`?

Reply:
Probably no difference.

Reply:
Actually [] is better. I forgot the developer looks at that code and [] is easier to understand.

Reply:
Actually the developer won't see  this code since it isn't inlined. Whatever. I switched to [] and [x] - looks simpler. Javascript interpreter probably treats these the same.

Reply:
@zth still sure you don't want to check in build artifacts?
These things would become transparent.